docs/src/site/optics.md

```diff
@@ -6,5 +6,12 @@ section: "optics"
 
 # Optics
 
+## Convert between two types
```

Comment:
At some point (0.5 or 0.6), optics were actually inheriting from each other. We removed it when we started to encode them as case class, nowadays I am not sure if we should use or not inheritance. I